### ISSUE TITLE
security/acme-client: dynu dns-01 validation method not working

### ISSUE THREAD

**Issue by meimi039** [opnsense/plugins#481]
Hello all!

This ist my first posting on github...

I noticed, that when I enter my Client ID and Secret for my ddns-Provider dynu.com into the validation-method page, these crendentials are not remembered.

In the logfile the following entries are found:

```
[Fri Jan 12 09:40:15 UTC 2018] Dynu client id and secret is not specified.
[Fri Jan 12 09:40:15 UTC 2018] Please create you API client id and secret and try again.
[Fri Jan 12 09:40:15 UTC 2018] Error add txt for domain:_acme-challenge.mydomain.dynu.com

```

Also in the /conf/config.xml there is not even an entry referring to dynu.com, except from the provider-pulldown menu.
I think the part is missing, where it stores the id and secret within the config.

Hope I didnt miss anything.

Greetings

Michael

**Comment by fichtner** [comment]
@meimi039 thank you for your report :) I've assigned the maintainer of the plugin

**Comment by fraenki** [comment]
I forgot to add several fields to the mode while implementing #213. This will be fixed in version 2.13 of our acme plugin (my attempt at numerology, ha!). Thanks again for your report!
